jenkins-job-components/src/main/groovy/com/ofg/job/component/general/ConcurrentJob.groovy

```diff
@@ -10,17 +10,13 @@ import javaposse.jobdsl.dsl.Job
  */
 @CompileStatic
 class ConcurrentJob implements JobComponent<Job> {
-
+    
```

Comment:
Please remove those extra spaces (there seems to be something wrong with your formatter).